web/static/js/components/css_modules/group_label_input.css

```diff
@@ -33,6 +33,24 @@
   }
 }
 
+.update-succeeded-checkmark {
+  animation: fadeOut ease 2s;
+  -webkit-animation: fadeOut ease 2s;
+  -moz-animation: fadeOut ease 2s;
+  -o-animation: fadeOut ease 2s;
+  -ms-animation: fadeOut ease 2s;
```

Comment:
Thank you for adding these vendor prefixes, @JennMiller and @ajoseph-stride, but we can kill them, as we use a webpack plugin called `autoprefixer` that augments the shorthand declaration—in this case, `animation`—with all revelant vendor prefixes.

Reply:
Thanks, didn't know that.  Modifying as per suggestion.